raddb/sites-available/abfab-tls

```diff
@@ -52,6 +89,11 @@ clients radsec-abfab {
 		proto = tls
 	}
 
+        client default_ip6 {
+		ipaddr = ::/0
```

Comment:
`ipv6addr` instead of `ipaddr`

right @alandekok ?

Reply:
No, ipaddr works.  It figures out the type of IP address from the formatting.

The only time you really need to use ipv4addr or ipv6addr is when you want to control whether the A or AAAA is used as the result of DNS resolution.